src/Main.cpp

```diff
@@ -12,24 +14,30 @@ struct Movement
 	geometry::Vec2 force;
 	F32            moveforce;
 
-	Movement(F32 moveforce, F32 velocitymax) : velocity(0.0f, 0.0f), moveforce(moveforce), velocitymax(velocitymax), force(0.0f, 0.0f) { }
+	Movement(F32 moveforce, F32 velocitymax) 
+		: velocity(0.0f, 0.0f)
+		, moveforce(moveforce)
+		, velocitymax(velocitymax)
+		, force(0.0f, 0.0f) 
+	{ 
+
+	}
 };
 
 struct KeyBindings 
 {
 	KeyCode left, down, up, right;
+	KeyCode normal, fox_eyes, scorpion_eyes;
```

Comment:
Why do we use a different key for every kind of eye? Can't we use the same key and produce a different result depending on context?